### ISSUE TITLE
Cannot read property 'addEventListener' of null

### ISSUE THREAD

**Issue by nm-a** [TypeCtrl/ngx-headroom#30]
Hi,
Not able to read the parent element, with "parent" input.

`
<ngx-headroom duration="500" easing="ease-in-out" [parent]="onGetParent" pinStart="50">
`

--onGetParent code
`
  public onGetParent(): any {
    //debugger;
    return document.getElementById("centreImage");
  }
`

Can you please help me in this.

thank you

**Comment by scttcper** [comment]
i think i left that in the docs on accident, it isn't implemented

**Comment by scttcper** [comment]
it might still be possible though let me check

**Comment by scttcper** [comment]
can try 
```ts
onGetParent = () => document.getElementById("centreImage");
```

**Comment by nm-a** [comment]
hi @scttcper , I am getting same error. This is happening because elements will be accessible only ngOnInit. This could be the reason for this error.

thank you

**Comment by scttcper** [comment]
yeah i think your mileage might really vary on this feature. I'm not sure how the react version handled it.
